src/impl/generated_specializations_cpp/change

```diff
@@ -0,0 +1,36 @@
+#! /usr/bin/env python
```

Comment:
I don't write python code often, wanted to check if this is python 2 vs 3 agnostic?